spec/features/hello_world_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+feature 'routing is working properly' do
```

Comment:
also, just for consistency can you do `describe` blocks instead of `feature` and `scenario`?